test/e2e/quickstart/test.sh

```diff
@@ -6,9 +6,13 @@ models_release="kubeai-models"
 
```

Comment:
The test for this functionality should be in its own e2e test, not the quickstart test... This rollout is not conducted in the quickstart.

Reply:
I think it makes sense to extend tests. The e2e test for quickstart is also testing the quickstart steps.

There is a limit on how many parallel tests we can execute. Quickstart made the most sense to me. In a follow up PR I'm extending the test for cacheprofile to also test mutating the URL.

Reply:
I basically don't want to replicate the quickstart in another test just so I can test mutating the url field.